Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
What do you mean by "regenerating the types"? May i ask why it is difficult to regenerate if we move it to thirdparty?

Staging uses the simlink trick to appear as vendored. I prefer not using staging if a plain directory like thirdparty/ works.

BTW, do you know if the clientset is generated in k8s.io/metrics? I tried to run client-gen and the output is quite different.